Makefile

```diff
@@ -48,15 +49,18 @@ lint: $(INSTALL_STAMP)
 	bin/lint.sh
 
 .PHONY: start
-start:
+start: $(INSTALL_STAMP) $(DOTENV_FILE)
 	poetry run python -m src.app.api
 
+$(DOTENV_FILE):
+	cp -n config/local_dev.env $(DOTENV_FILE)
```

Comment:
maybe we should move this `local_dev.env` file to the root level and use the `.env.example` convention to make what's happening here a little more obvious.